build-tools/xa-prep-tasks/Xamarin.Android.BuildTools.PrepTasks/SystemUnzip.cs

```diff
@@ -166,10 +166,10 @@ public override bool Execute ()
 
 		IEnumerable<string> GetExtractedSourceEntries (string root)
```

Comment:
Please rename this method to `GetExtractedSourceDirectories()`, as it now only returns directories, not file system entries.